### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by dipdee**

### CODE REVIEW

**Comment by dipdee** [comment]
@tushenkaman
@netflash

**Comment by netflash** [comment]
why don't you use the "reviewers" tab ?

File: udacity/log-analysis-project/Readme.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
-You will need to unzip **newsdata.sql.zip** file after downloading it. The file inside is called **newsdata.sql** Put this file into the [vagrant](https://www.vagrantup.com/downloads.html) directory, which is shared with your virtual machine.
+In this project we work with the database.The database contains newspaper articles, as well as the web server log for the site. The log has a database row for each time a reader loaded a web page. Using that information, code answer questions about the site's user activity.
+
+You need to use a virtual machine (VM) to run an SQL database server.
+
+VirtualBox is the software that actually runs the virtual machine. You can download it from [HERE](virtualbox.org). Install the platform package for your operating system.
+
+Vagrant is the software that configures the VM and lets you share files between your host computer and the VM's filesystem. Download it from [vagrantup.com](https://www.vagrantup.com/downloads.html).
```
Comment (netflash) [inline udacity/log-analysis-project/Readme.md:7]: Virtualbox itself handles the sharing, not Vagrant.

File: udacity/log-analysis-project/reporting.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```
Comment (netflash) [inline udacity/log-analysis-project/reporting.py:1]: 2 or 3 ?

File: udacity/log-analysis-project/reporting.py [hunk 3]
```diff
@@ -39,16 +45,19 @@ def log_errors():
...
     datb = psycopg2.connect(database='news')
     conn = datb.cursor()
-    conn.execute("""select date, round((100* sum / (SUM(sum) OVER ())),1) "% of errors", status from error_log2;""")
+    conn.execute("""select date, ROUND((100* sum / (SUM(sum) OVER ())),1),status"
+                    " from total_log order by round desc;""")
     result = conn.fetchall()
+    print '\n' + "Days when requests lead to errors:" + '\n'
     for output in result:
-        print "On the %s were %s%% requests lead to errors" % (output[0], output[1])
+# I used output instead of date and round, because round is a build-in function.
```
Comment (netflash) [inline udacity/log-analysis-project/reporting.py:53]: `builT-in`

**Review by netflash — COMMENTED** [review]
Have just skimmed that.
Overall looks good.

File: udacity/log-analysis-project/Readme.md (hunk 1 above)
Comment (dipdee) [reply udacity/log-analysis-project/Readme.md:7]: Noted!

File: udacity/log-analysis-project/reporting.py (hunk 3 above)
Comment (dipdee) [reply udacity/log-analysis-project/reporting.py:53]: OK!

File: udacity/log-analysis-project/reporting.py (hunk 2 above)
Comment (dipdee) [reply udacity/log-analysis-project/reporting.py:1]: Excuse me, could you explain what do you mean?

**Pull request closed without merging**